TestProjects/UniversalGraphicsTest/Assets/CommonAssets/Scripts/ChangeShadowCascadeSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+public class ChangeShadowCascadeSettings : MonoBehaviour
```

Comment:
If it's public it needs API docs, otherwise make internal

Reply:
This is a test script for the URP project so I assume we don't need to document it :)

Reply:
yupp test script